### PR TITLE
Add reinstall kernel notebook

### PR DESCRIPTION
**Pull request by HumairAK**
Related: https://github.com/operate-first/support/issues/148

### CODE REVIEW

File: docs/userstories/jupyterhub/reinstall_kernel.feature [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+        Given I am a user of MOC-ZERO
+        * I have access to JupyteHub on MOC-ZERO
+        * I have logged in to a Jupyter Notebook with a terminal open
+
+    Scenario: Restart Kernel for Jupyter Notebook
+
+        This scenario is based on [Issue 243](https://github.com/operate-first/support/issues/142#issuecomment-806220803).
+
+        When I retrieve the KERNEL_TYPE and DISPLAY_NAME by running `jupyter kernelspec list --json`
+        * I run command `ipython kernel install --user --name=${KERNELTYPE} --display-name '${DISPLAY_NAME}'
```
Comment (anishasthana) [inline docs/userstories/jupyterhub/reinstall_kernel.feature]: Should there be another '`' at the end of this line?

**Review by anishasthana — CHANGES_REQUESTED** [review]

File: docs/userstories/jupyterhub/reinstall_kernel.feature [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+        Then the chosen Kernel should be restarted
```
Comment (tumido) [inline docs/userstories/jupyterhub/reinstall_kernel.feature]: The kernel is not restarted, it's reinstalled.. JupyterHub/Lab will then restart the Kernel.. maybe we should be precise on that.. What about something like:

```suggestion
        Then the Kernel is restored and ready to be started by JupyterHub again
```

**Review by tumido — COMMENTED** [review]
/lgtm

**Review by tumido — APPROVED** [review]
/approve

**Pull request merged**